docs/changehistory/NextVersion.md

```diff
@@ -102,3 +102,9 @@ In addition, new protected static methods were added:
 The following method is now `async` to make it easier to integrate with asynchronous status and health reporting services:
 
 * [IModelExportHandler.onProgress]($backend)
+
+### @bentley/ecschema-metadata package
+
+Properties getter in [ECClass]($ecschema-metadata) have been changed to return an iterator of properties instead of an array of properties.
+Array indexing and properties like .length will no longer work with the returned iterator, so you may need to create an array from the iterator or use its .next() method. Iterating with for...of loop works the same with iterator as before with an array.
+This change is made because internally properties are now stored in a map instead of an array, and it is more efficient to return an iterator for the properties to be generated on demand than to create them on the getter.
```

Comment:
Can you emphasize this is a `@beta` API? Don't want people reading change notes to think we are breaking `@public` APIs.

Reply:
Added `@beta` before [ECClass]